src/copyCodeBlock.js

```diff
@@ -21,12 +23,12 @@ export default (string, opts) => {
                     textarea.select();
 
                     button.innerHTML = 'Copied!';
-                    setTimeout(function() { button.innerHTML = 'Copy'; }, 1000);
+                    setTimeout(function() { button.innerHTML = 'Click to copy'; }, 1500);
```

Comment:
Might be a good idea to break `'Click to copy'` into its own variable so it can be reused.